cmd/environment.go

```diff
@@ -36,6 +36,7 @@ func New() *EnvSettings {
 // AddBaseFlags binds base flags to the given flagset.
 func (s *EnvSettings) AddBaseFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.DryRun, "dry-run", false, "simulate a command")
+	
```

Comment:
I guess this file does not need to change?

Reply:
correct removed blank lines

Reply:
fixed